tests/functional/testSubscription.js

```diff
@@ -1486,10 +1483,125 @@ describe('Subscription view', function() {
+           };
+         }
+
+         await openSubscriptionManagement();
+       });
+
+        it('accountScreen impression event is recorded', async () => {
+
+          //Open the subscription management view and record accountScreen telemetry
+          const accountScreenEvents = await vpn.gleanTestGetValue("impression", "accountScreen", "main");
+          assert.equal(accountScreenEvents.length, 1);
+          const accountScreenEventExtras = accountScreenEvents[0].extra;
+          assert.equal(subscriptionManagementScreenTelemetryId, accountScreenEventExtras.screen);
+        });
+
+        it('editSelected interaction event is recorded', async () => {
+
+          //Click the user profile button and record editSelected telemetry
+          await vpn.waitForQueryAndClick(queries.screenSettings.subscriptionView.SUBSCRIPTION_USER_PROFILE_BUTTON_ACCOUNT.visible());
+          const editSelectedEvents = await vpn.gleanTestGetValue("interaction", "editSelected", "main");
+          assert.equal(editSelectedEvents.length, 1);
+          const editSelectedEventsExtras = editSelectedEvents[0].extra;
+          assert.equal(subscriptionManagementScreenTelemetryId, editSelectedEventsExtras.screen);
+        });
+
+        if (isMonthlyPlanTest) {
+          it('changePlanSelected interaction event is recorded with correct screen', async () => {
+
+            //Click the change plan button and record changePlanSelected telemetry
+            await vpn.waitForQueryAndClick(queries.screenSettings.subscriptionView.ANNUAL_UPGRADE_BUTTON.visible());
+            const changePlanSelectedEvents = await vpn.gleanTestGetValue("interaction", "changePlanSelected", "main");
+            assert.equal(changePlanSelectedEvents.length, 1);
+            const changePlanSelectedEventsExtras = changePlanSelectedEvents[0].extra;
+            assert.equal(subscriptionManagementScreenTelemetryId, changePlanSelectedEventsExtras.screen);
+          });
+        }
+
+        it('manageSubscriptionSelected interaction event is recorded', async () => {
+
+          //Click the manage subscription button and record manageSubscriptionSelected telemetry
+          await vpn.scrollToQuery(queries.screenSettings.subscriptionView.FLICKABLE, queries.screenSettings.subscriptionView.SUBSCRIPTION_USER_PROFILE_BUTTON_SUB.visible());
+          await vpn.waitForQueryAndClick(queries.screenSettings.subscriptionView.SUBSCRIPTION_USER_PROFILE_BUTTON_SUB.visible());
+          const manageSubscriptionSelectedEvents = await vpn.gleanTestGetValue("interaction", "manageSubscriptionSelected", "main");
+          assert.equal(manageSubscriptionSelectedEvents.length, 1);
+          const manageSubscriptionSelectedEventsExtras = manageSubscriptionSelectedEvents[0].extra;
+          assert.equal(subscriptionManagementScreenTelemetryId, manageSubscriptionSelectedEventsExtras.screen);
+        });
+
+        it('homeSelected interaction event is recorded with correct screen', async () => {
```

Comment:
@brizental do you think it is worth testing the navbar telemetry with the appropriate `screen` extra key like I did here for every different potential screen? Or should we just test it once somewhere else?